spec/controllers/grade_entry_forms_controller_spec.rb

```diff
@@ -342,4 +342,23 @@
       end.to change { ActionMailer::Base.deliveries.count }.by(1)
     end
   end
+
+  describe '#student_interface' do
+    before :each do
+      allow(controller).to receive(:session_expired?).and_return(false)
```

Comment:
You don't need this one or the next line (they just repeat the previous mocking). Only the `:current_user` mock needs to change.